subsys/bluetooth/host/Kconfig

```diff
@@ -4,14 +4,37 @@
 # Copyright (c) 2015-2016 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+config BT_LONG_WQ
+	bool "Dedicated workqueue for long-running tasks."
+	default y if BT_GATT_CACHING || BT_TINYCRYPT_ECC
```

Comment:
Does `BT_TINYCRYPT_ECC` work without this? 

i.e. should we have a `select` or `depends on` for `BT_TINYCRYPT_ECC`?

Reply:
I'll test that next week yeah. In theory it should work, since it'll just get submitted to the sys wq, but the stack size is not propagated there, and I just suspect bad behavior all around like johan said in another comment. If it doesn't work, I'll add a `select` or `depends on`, I always forget which one's the strongest.

Reply:
`select` and `depends on` are equally strong :) You may be thinking of `default` or `imply` which can be overridden.